src/main/java/org/worshipsongs/importer/SongDao.java

```diff
@@ -59,8 +67,7 @@ public boolean insertSong(Connection connection, Song song, SongBook songBook)
             preparedStatement.executeUpdate();
             preparedStatement.close();
             return true;
-        }
-        catch (Exception e) {
+        } catch (Exception e) {
```

Comment:
Daos shouldn't catch exception. There is no point returning a true or false here. Instead it would be better to just throw the exceptions.